deploy/scripts/src/ipv-core/test.ts

```diff
@@ -294,12 +367,13 @@ export function coreScenario1 (): void {
   sleep(Math.random() * 3)
 
   group(
-    'B01_Core_06_AddrDataContinue POST',
+    'B01_Core_07_AddrDataContinue POST',
     function () {
       const startTime1 = Date.now()
       res = http.post(
         addressStubURL,
         {
+          sort: 'Kenneth Decerqueira (Valid Experian) Address',
```

Comment:
```suggestion
          sort: user.address,
```
Suggestion: Paramaterise these values

Reply:
Done